test/utils/base/cluster_test_runner.go

```diff
@@ -47,21 +45,17 @@ type ClusterTestRunnerBase struct {
 	unitTestMock      bool
 }
 
-// Initialize parses the command line arguments
-func (c *ClusterTestRunnerBase) Initialize(m *testing.M) {
-	// Parsing flags
-	ParseFlags(m)
-	// Running tests
-	os.Exit(m.Run())
-}
-
-func (c *ClusterTestRunnerBase) Build(t *testing.T, needs ClusterNeeds, vanClientProvider VanClientProvider) []*ClusterContext {
+func (c *ClusterTestRunnerBase) BuildOrSkip(t *testing.T, needs ClusterNeeds, vanClientProvider VanClientProvider) []*ClusterContext {
 
 	// Initializing internal properties
 	c.T = t
 	c.vanClientProvider = vanClientProvider
 	c.ClusterContexts = []*ClusterContext{}
 
+	if len(TestFlags.KubeConfigs) == 0 && len(TestFlags.EdgeKubeConfigs) > 0 {
+		panic("at least one --kubeconfig must be provided when using --edgekubeconfig")
+	}
+
```

Comment:
What is the benefit of moving this logic here instead of at the `ParseFlags` function?

Reply:
I seemed to me ... that when you do "parseFlags" you are not expecting to receive a panig because of "flags" interpretation.... 
you just want to parse...
anyway, it is true that probably is a change not related, and maybe not necessary, I can roll it back if you prefer

Reply:
It makes more sense to me to keep it there, because the `Build` method can be called multiple times,
at the package level, while it just needs to be done once.

Reply:
Sure! restoring to its place